dojo/settings/settings.dist.py

```diff
@@ -823,6 +823,7 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
     'DSOP Scan': True,
     'Acunetix Scan': True,
     'Trivy Scan': True,
+    'GitLab Dependency Scanning Report': True,
```

Comment:
i think this one can now be removed if cwe is not part of the hash calculation.

Reply:
done! thanks for spotting that 👍🏻